test/integration/test_integration.py

```diff
@@ -180,30 +183,23 @@ def assert_sample_comparison(sample_simple, sample_list, mongo_client, referenci
             mongo_client["HealthcareService"].find_one(
                 {"identifier": {"$elemMatch": {"value": val}}}, ["id"]
             )
-            for val in ["15067", "15068", "15069", "15070"]
+            for val in vals_list
         ]
 
-    assert sample_simple == {
+    expected_simple = {
         "_id": sample_simple["_id"],
         "id": sample_simple["id"],
         "resourceType": "Patient",
-        "language": "Engl",
+        "language": {"coding": [{"code": "ENGL"}]},
         "identifier": sample_simple["identifier"],
         "gender": "female",
         "birthDate": "1844-07-18",
-        "address": [
-            {"city": "Paris", "country": "France"},
-            {"city": "NY", "state": "NY state", "country": "USA"},
-        ],
-        "generalPractitioner": [
-            {
-                "identifier": {
-                    "system": "HealthcareService",
-                    "value": ref_simple["id"] if referencing_done else "48902",
-                }
-            }
-        ],
+        "address": {"city": "Paris", "country": "France"},
```

Comment:
why so we have a single address now ?

Reply:
This comes from the structure definition.
But you're right, maybe we should still have an array here... Do you think we could have parsed this one wrong?

Reply:
Ok now we know it's because of the backbone element bug, can we wait to fhix it before we merge this PR ? I'll sleep better at night.